### PR TITLE
LPS-117778 Show ItemConfiguration

### PR DESCRIPTION
**Pull request by p2kmgcl**

### CODE REVIEW

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/actions/setFragmentEditables.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import {SET_FRAGMENT_EDITABLES} from './types';
+
+export default function moveItem(fragmentEntryLinkId, editables) {
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/actions/setFragmentEditables.js:17]: Rename fn

File: modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentFloatingToolbar.js [hunk 2]
```diff
@@ -12,144 +12,16 @@
...
-			editableId: PropTypes.string.isRequired,
-			editableValueNamespace: PropTypes.string.isRequired,
-			element: PropTypes.object.isRequired,
-			processor: PropTypes.object,
-		})
-	),
-	fragmentEntryLinkId: PropTypes.string.isRequired,
-	onButtonClick: PropTypes.func.isRequired,
-};
+/**
```
Comment (p2kmgcl) [inline modules/apps/layout/layout-content-page-editor-web/src/main/resources/META-INF/resources/page_editor/app/components/fragment-content/FragmentContentFloatingToolbar.js:15]: Delete file

**Comment by sandrodw3** [comment]
https://github.com/liferay-echo/liferay-portal/pull/1655

**Comment by p2kmgcl** [comment]
https://github.com/sandrodw3/liferay-portal/pull/356

**Pull request closed without merging**